src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -79,10 +96,8 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
 
   const weekdays = useMemo(() => {
-    const today = startOfToday();
-
     return range(0, SCHEDULE_SHOW_DAYS_AHEAD).map((dayIndex) => {
-      const day = addDays(today, dayIndex);
+      const day = addDays(firstDay, dayIndex);
```

Comment:
```suggestion
      const day = addDays(firstDayOfSchedule, dayIndex);
      const formatDayLabel = (day: Date | number) => {
        if (isScheduleTimeshifted) {
          return format(day, "E, LLL d");
        } else {
          return dayIndex === 0 ? "Today" : format(day, "E");
        }
      }
```

Reply:
Done

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef

Reply:
Actually, looking at the code suggestion above (+ in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef#r635716283) vs what is implemented in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef, it looks like you didn't end up going with the `isScheduleTimeshifted` implementation.

Why was that?